ffig/filters/capi_filter.py

```diff
@@ -51,7 +51,7 @@ def to_go(t):
         if t.pointee.kind == TypeKind.CHAR_S:
             return 'string'
         if t.pointee.kind == TypeKind.RECORD:
-            return 'unsafe.Pointer'
+            return t.pointee.name.replace('const', '')
```

Comment:
On reflection, matching `const` will hit `constantFoo`. Matching `const_` will not.
Ideally we'd get rid of the need for any of this `const` manipulation by getting the info from libclang. We might need to extend libclang to do so.